src/resources/startup.py

```diff
@@ -128,7 +131,7 @@ def deserialize_bytes_tensor(encoded_tensor):
         sb = struct.unpack_from("<{}s".format(l), val_buf, offset)[0]
         offset += l
         strs.append(sb)
-    return (np.array(strs, dtype=bytes))
+    return (np.array(strs, dtype=np.bytes_))
```

Comment:
We need to use np.object here to avoid removal of zero bytes. Also in the description we should state that the returned numpy array will have elements of type np.object.

Reply:
I will fix this in a subsequent PR. This is only to ensure that `np.object_` is parsed correctly.